src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -63,9 +88,30 @@ const BudgetAssignmentsTable = ({
     itemCount={tableData?.count || 0}
     pageCount={tableData?.numPages || 1}
     EmptyTableComponent={CustomDataTableEmptyState}
+    bulkActions={[
+      <RemindAction />,
+      <CancelAction />,
+    ]}
+    onSelectedRowsChanged={() => {}}
   />
 );
 
+CancelAction.propTypes = {
+  selectedFlatRows: PropTypes.arrayOf(PropTypes.shape()),
```

Comment:
nit: may be helpful to include the contents of the `PropTypes.shape()`.

Reply:
> Related, if learnerEmail isn't actually used within this component (same with contentKey), should they be excluded from the prop types for this component?
I would say that this applies here as well? Currently selectedFlatRows isn't used for anything but the length of the array and logging the contents for debugging purposes.

Reply:
It's likely more important once the integration with the API occurs and the `selectedFlatRows` are iterated, etc. Totally cool to defer.